android-core/src/main/java/com/mparticle/modernization/kit/MParticleKit.kt

```diff
@@ -0,0 +1,33 @@
+package com.mparticle.modernization.kit
+
+import com.mparticle.BaseEvent
+import com.mparticle.modernization.eventlogging.MParticleEventLogging
+import java.math.BigDecimal
+
+abstract class MParticleKit : MParticleEventLogging {
```

Comment:
Is there an mParticle Kit equivalent that includes the other non-event logging methods?  (onLogin, onSetUserAttribute, etc)

Reply:
I think this is probably just missing the addition of the other interfaces and that actually it should be implementing multiple interfaces, not only the event logging one.

Reply:
Those are either listener interfaces or internal interfaces.. MParticleEventLogging is the api providing access to logEvent and those kind of things. Adding to it multiple interfaces can be implemented by this or any other class if needed.